---a/net/minecraft/block/BlockGlass.java
+++b/net/minecraft/block/BlockGlass.java
@@ -1,39 +1,29 @@
 package net.minecraft.block;
 
 import java.util.Random;
+
 import net.minecraft.block.material.Material;
 import net.minecraft.creativetab.CreativeTabs;
-import net.minecraft.util.EnumWorldBlockLayer;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class BlockGlass extends BlockBreakable
-{
+
+public class BlockGlass extends BlockBreakable {
+
     private static final String __OBFID = "CL_00000249";
 
-    public BlockGlass(Material p_i45408_1_, boolean p_i45408_2_)
-    {
+    public BlockGlass(Material p_i45408_1_, boolean p_i45408_2_) {
         super(p_i45408_1_, p_i45408_2_);
         this.setCreativeTab(CreativeTabs.tabBlock);
     }
 
-    public int quantityDropped(Random random)
-    {
+    public int quantityDropped(Random random) {
         return 0;
     }
 
-    @SideOnly(Side.CLIENT)
-    public EnumWorldBlockLayer getBlockLayer()
-    {
-        return EnumWorldBlockLayer.CUTOUT;
-    }
-
-    public boolean isFullCube()
-    {
+    public boolean isFullCube() {
         return false;
     }
 
-    protected boolean canSilkHarvest()
-    {
+    public boolean canSilkHarvest() { // SpigotMCP -- Make public as bukkit assumes in the same package
         return true;
     }
-}
+}
